save-core/src/nativeMain/kotlin/org/cqfn/save/core/utils/ProcessBuilderInternal.kt

```diff
@@ -0,0 +1,35 @@
+@file:Suppress("FILE_WILDCARD_IMPORTS")
+
+package org.cqfn.save.core.utils
+
+import org.cqfn.save.core.logging.logDebug
+
+import okio.FileSystem
+import okio.Path
+import platform.posix.system
+
+@Suppress("MISSING_KDOC_TOP_LEVEL",
+    "MISSING_KDOC_CLASS_ELEMENTS",
+    "MISSING_KDOC_ON_FUNCTION"
+)
+actual class ProcessBuilderInternal actual constructor(
+    private val stdoutFile: Path,
+    private val stderrFile: Path,
+    private val collectStdout: Boolean
+) {
+    actual fun prepareCmd(command: String) = if (collectStdout) {
+        "$command >$stdoutFile 2>$stderrFile"
+    } else {
+        "$command 2>$stderrFile"
+    }
+
+    actual fun exec(cmd: String): Int {
+        logDebug("Executing: $cmd")
```

Comment:
I think this logging should be performed in `ProcessBuilder` before calling `exec`, so that it will be done for all platforms